### ISSUE TITLE
How Can I run it in mutiple machines?

### ISSUE THREAD

**Issue by younfor** [liangfengsid/tensoronspark#4]
I find that the code just run once on a machines? 
should I configure some ip ???

**Comment by liangfengsid** [comment]
The code is run on top of Spark. You will first need to set up a Hadoop and Spark cluster, which can distribute jobs into multiple machines. Once you have a Spark cluster, you can run the code in one machine, which acts as a client, and the Tensorflow program will be automatically submit to the cluster and run in a distributed way. In other words, submit codes in one machines, run in multiple machines. 

You may need to configure the ip/host when you are setting up the Hadoop&Spark cluster, and you also need to put the training data in Hadoop (for distributed data) for your Tensorflow program to access the distributed data. 

For the installation of the Hadoop&Spark cluster, you can refer to [http://hadoop.apache.org](url) and [http://spark.apache.org](url). 
To look at a complete MNIST example of how to code via TensorSpark interfaces, you can refer to [https://github.com/liangfengsid/tensoronspark/blob/master/tensorspark/example/spark_mnist.py](url)

**Comment by younfor** [comment]
Thanks a lot , I got your point. 
Hope the distributed dl with spark will popular.
it will be better if given some example of cifar-10 cifar-100 or even imagenet  , and benchmark of time, accuracy and so on

**Comment by liangfengsid** [comment]
We appreciate your advice. We are working with effort to improve the project and verify the performance with more real dl applications.
